run

```diff
@@ -521,6 +529,12 @@ run_mkdrive() {
             local targetdir=$(realpath $5)
             local targetvol=(-v "$targetdir:$targetdir")
         fi
+        if [ "$(uname)" == "Darwin" ]; then
+            docker run --rm                        \
+                -v /lib/modules:/lib/modules       \
```

Comment:
I understand intention here but I doubt the volumes should be passed.

Reply:
If not mount a volume with modules then:
docker run $DOCKER_IMAGE modprobe loop
modprobe: FATAL: Module loop not found in directory /lib/modules/4.12.14-25.28-default